src/Business Foundation/App/NoSeriesCopilot/src/Copilot/NoSeriesCopilotImpl.Codeunit.al

```diff
@@ -0,0 +1,518 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+using System.Telemetry;
+using System.AI;
+using System.Text.Json;
+
+codeunit 324 "No. Series Copilot Impl."
+{
```

Comment:
"Impl." codeunits should always have Access = Internal. 
If some of this should be public you should add a facade codeunit.